layers/build-krd/dracut/soci/soci-settled.sh

```diff
@@ -87,12 +87,15 @@ wait_on_zot() {
 }
 
 start_zot() {
+    local xdgcfghome="/root/.zot-xdg-config-home"
     cat > /etc/systemd/system/soci-zot.service << EOF
 [Unit]
 Description=Start zot for soci mount
 
 [Service]
 Type=simple
+Environment=XDG_CONFIG_HOME=$xdgcfghome
+ConfigurationDirectory=$xdgcfghome
```

Comment:
What are the implications of setting ConfigurationDirectory ?  systemd.exec manpage says it defaults to /etc.

Reply:
I don't know. it defaults to /etc/ for system units (of which this is one), but $XDG_CONFIG_HOME for user-units.

I'm not sure of the implications, but i wanted to get the directory made and this seems like the "proper" way to have systemd make directories for a service.

Reply:
ok let's just try it, thanks.